include/zephyr/drivers/dma/dma_rpi_pico.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2023 TOKITA Hiroshi <tokita.hiroshi@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DRIVERS_DMA_RPI_PICO_H_
+#define ZEPHYR_INCLUDE_DRIVERS_DMA_RPI_PICO_H_
+
+#include <zephyr/device.h>
+#include <zephyr/dt-bindings/dma/rpi_pico_dma.h>
+
+/**
+ * Set DMA priority to high
+ *
+ * It should call after the dma_config().
+ *
+ * @param dev DMA device
+ * @param ch Channel to applicate
+ * @param hpri true for high-proirity
+ * @return 0 on success, negative value if errored
+ */
+extern int rpi_pico_dma_channel_set_high_priority(const struct device *dev, uint32_t ch, bool hpri);
+
+
+/**
+ * Configure rpi_pico's DMA specific feature.
+ *
+ * This function handles the `channel-config` cell value.
+ * It should call after the dma_config().
+ *
+ * @param dev DMA device
+ * @param ch Channel to applicate
+ * @param ch_cfg The value of `channel-config` cell
+ * @return 0 on success, negative value if errored
+ */
+extern int rpi_pico_dma_channel_config(const struct device *dev, uint32_t ch, uint32_t ch_cfg);
```

Comment:
what is ch_cfg in this case? If we need to expand the dma_config struct to cover this, that seems more sensible that adding rpi specific functions if possible.

Reply:
Since it is not necessary for basic operation, I deleted it.
For functions that may be able to common (For example, rpi_pico has a function to change the order of 2 bytes and 4 bytes), I will consider to create as a separately PR if necessary.